app/AccountService.js

```diff
@@ -0,0 +1,38 @@
+var allUsersList = JSON.parse(usersJSON);
+
+var email;
+var password;
+
+function LoggedIn(){
+  alert('Logged in!');
+
+  // var fso = new ActiveXObject("Scripting.FileSystemObject");
+  // var a = fso.CreateTextFile("/home/vlad.mararu/work/cluj-pre-js-2016/app/test.txt", true);
+  // a.WriteLine('User logeed in now!');
+  // a.Close();
+}
+
+function LoggedOut(){
+  user1.setLogged(false);
+  alert('Logged out');
+}
+
+function checkLogin(){
+  email = document.getElementById('email').value;
+  password = document.getElementById('password').value;
+
+  var user1 = new User(email, password);
+
+
+  for(var i = 0; i < allUsersList.length; i++){
```

Comment:
you should do the check against a collection of User objects. don't use anymore any JSON data after they've been parsed and User, Song etc objects have been created